dts/arm/st/f4/stm32f429Xi.dtsi

```diff
@@ -24,4 +25,44 @@
 			};
 		};
 	};
+
+	sdram2: sdram@d0000000 {
```

Comment:
This needs to be added at board level, RAM size can't be specified at SoC level as it's an arbitrary value

Reply:
Makes sense.